regression/cbmc-incr-oneloop/ignore-before-unwind/main.c

```diff
@@ -0,0 +1,14 @@
+int main()
+{
+  __CPROVER_bool assumption = (__CPROVER_bool)0;
+  __CPROVER_bool property = (__CPROVER_bool)1;
+
+  while(1)
+  {
+    __CPROVER_assume(property == 1);
```

Comment:
What is the purpose of the unused `assumption` variable?

Reply:
Hard to say, but if you comment it out, it suddenly starts to work.

Reply:
I mean the assumption variable, not the assumption statement.

Reply:
Sorry, yes, that one is irrelevant.